proto/storage/alert.proto

```diff
@@ -34,9 +34,14 @@ message Alert {
     // Represents an alert on a kubernetes resource (configmaps, secrets, etc.)
     message Resource {
         enum ResourceType {
-            UNKNOWN     = 0;
-            SECRETS     = 1;
-            CONFIGMAPS  = 2;
+            UNKNOWN                    = 0;
```

Comment:
I believe we are purposeful to not include deployment. Might be worth calling out.